inductiva/tasks/streams.py

```diff
@@ -1,33 +1,63 @@
-"""Module for consuming the streams of a task through a websocket."""
+"""Module for consuming the streams of a task through a websocket.
+
+Task logs are output in the following format:
+
+$ induction logs <task_id>
+<log line 1>                                        log content
+<log line 2>
+  ...
+<log line N>
+                                                    `SPACING` black lines
+ ● 0:00:02 Streaming output from task <task_id>     connection status footer
+"""
+from threading import Thread, RLock
+from datetime import timedelta
 from typing import IO
 import logging
+import sched
 import json
 import sys
+import time
+import os
 
 import websocket
 
 import inductiva
 from inductiva import constants
 
 logger = logging.getLogger("websocket")
-logger.setLevel(logging.DEBUG)
+logger.setLevel(logging.ERROR)
+
+SPACING = 1
+ESC = "\u001B["
+RESET = f"{ESC}0m"
```

Comment:
Why not move this to the enum class under [`inductiva/utils/format_utils.py`](https://github.com/inductiva/inductiva/blob/96016a4e739feba43b215900661343cd6a16f7d3/inductiva/utils/format_utils.py#L28) and then call the function [`emphasis_formatter`](https://github.com/inductiva/inductiva/blob/96016a4e739feba43b215900661343cd6a16f7d3/inductiva/utils/format_utils.py#L60)